### ISSUE TITLE
`gvar.var` fails on non-gvar arrays longer than some threshold

### ISSUE THREAD

**Issue by Gattocrucco** [gplepage/gvar#31]
`gvar.var` works for non-gvar numbers, returning 0. However it misteriously fails on an array long enough:

```python
import numpy as np
import gvar
print(gvar.__version__)

for n in range(100):
    try:
        gvar.var(np.ones(n))
    except Exception as exc:
        print('failed at n =', n)
        raise exc
```

```
11.10.1
failed at n = 51
Traceback (most recent call last):
  File "<stdin>", line 6, in <module>
  File "<stdin>", line 3, in <module>
  File "_utilities.pyx", line 900, in gvar._utilities.var
ZeroDivisionError: float division
```

**Comment by gplepage** [comment]
This will be fixed in the next release.

**Comment by gplepage** [comment]
Fixed in v11.11. Thanks again for pointing out the problem.

**Comment by Gattocrucco** [comment]
Thanks.
